UIKitDZ/UIKitDZ/PlayerViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  PlayerViewController.swift
+//  UIKitDZ
+//
+//  Created by Дмитрий Супрун on 25.09.22.
+//
+
+import AVFoundation
+import UIKit
+/// Main player with control elements
+final class PlayerViewController: UIViewController {
+    
+    // MARK: - IBOutlet
+    @IBOutlet weak var trackImageView: UIImageView!
+    @IBOutlet weak var playButton: UIButton!
+    @IBOutlet weak var trackSlider: UISlider!
+    @IBOutlet weak var timeFromBeginLabel: UILabel!
+    @IBOutlet weak var timeToEndLabel: UILabel!
+    
+    // MARK: - PublicProperties
+    var startWithTrack: (song: String, albumImage: String)!
+    
+    // MARK: - PrivateProperties
+    private var player = AVAudioPlayer()
+    private let media = [(song: "Peremen", albumImage: "LastHero"), (song: "WindOfChange", albumImage: "CrazyWord")]
+    
+    // MARK: - ViewControllers
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        playTrack(track: startWithTrack.song, image: startWithTrack.albumImage)
```

Comment:
```suggestion
        configureTrackView()
```
убрать в метод все из viewDidLoad

Reply:
Fixed it.